.github/workflows/build_formplayer.yml

```diff
@@ -0,0 +1,33 @@
+name: CommCare Core Builds for Formplayer
+
+on:
+  push:
+    branches: [formplayer]
+  pull_request:
+    branches: [formplayer]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.3
+          arguments: build
+      - name: Cli Jar
+        uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.3
+          arguments: cliJar
+      - name: Upload Archives
```

Comment:
What's the use case for storing the archives?

Reply:
Not sure, mostly adding it here since Jenkins build was storing it. Though I imagine the idea being if someone wants to use CLI directly without having to built FP locally, they can use these jars.

Reply:
I would punt this for now until we have a need.